### ISSUE TITLE
Some mathbase functions seem to have disappeared in v1.8

### ISSUE THREAD

**Issue by rossberg** [sphinx-doc/sphinx#6708]
**Describe the bug**
The v1.8 release deprecated `mathbase`, and it is scheduled for removal in v3.0.

However, we [recently discovered](https://github.com/WebAssembly/spec/issues/1075) that some functionality has already vanished in v1.8 itself (and later), e.g. `latex_visit_math` etc.

Moreover, is there any replacement for the `mathbase` API? The [WebAssembly specification](https://github.com/WebAssembly/spec/tree/master/document/core) relies [heavily](https://github.com/WebAssembly/spec/blob/master/document/core/util/mathdef.py) on `mathbase` to incorporate latex macros for formal stuff.

**To Reproduce**
Steps to reproduce the behaviour:
```
$ git clone https://github.com/WebAssembly/spec
$ cd spec/document/core
$ make html
```

**Expected behaviour**
Build succeeds.

**Your project**
https://github.com/WebAssembly/spec

**Additional context**
https://github.com/WebAssembly/spec/issues/1075

**Comment by tk0miya** [comment]
Sorry for inconvenience. I'll restore it in next release.

>Moreover, is there any replacement for the mathbase API? The WebAssembly specification relies heavily on mathbase to incorporate latex macros for formal stuff.

Now both `math` and `math_block` (called as `displaymath` in mathbase module) has became built-in node class (docutils has provided them since v0.8!). So all built-in writers supports them now. So you can call it via `self.visit_math(node)` or `self.visit_math_block(node)`.

**Comment by tk0miya** [comment]
I just posted #6720 to fix this. Please try it if you have time. It will be merged and released in next stable release.
Thanks,

**Comment by tk0miya** [comment]
Fixed by #6720. Thank you for reporting.
